spack/packages/fiber/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack import *
+
+FFT_LIBS = (
+    'heffte',
+    'fftw',
+    'ffte',
+)
+
+
+class Fiber(CMakePackage):
+    """High Performance Fast Fourier Transform benchmark."""
+
+    homepage = "https://fiber.icl.utk.edu/"
+    git      = "https://github.com/icl-utk-edu/fiber"
+
+    maintainers = ['G-Ragghianti', 'luszczek']
+
+    version('master', branch='master')
+
+    variant('fft', default='none', values=FFT_LIBS,
+            multi=True, description='Supported FFT libraries')
+    #variant('onlycomplex', default=False)
+
+    depends_on('mpi')
+
+    # Iterate over the fftlib variant for simple dependencies
+    for fft in FFT_LIBS:
+        depends_on(fft, when='fft={0}'.format(fft))
+
+    depends_on('heffte+fftw+cuda', when='fft=heffte')
```

Comment:
`depends_on('fftw', when'fft=fftw')` and so on. What if `fft=heffte,ffte`?

Reply:
Spack actually handles this correctly because it isn't doing a direct string comparison as you are thinking.  For mulit-value variants, it breaks apart the values before doing the check, so "fft=fftw" matches both the opvious case and also matches with "fft=asdf,fftw,sdfg".